components/Skills.jsx

```diff
@@ -53,14 +53,14 @@ const Skills = () => {
             <div className="grid grid-cols-2 gap-4 justify-center items-center">
               <div className="m-auto">
                 <img
-                  src="/skills/react.png"
-                  width="64px"
-                  height="64px"
+                  src="/skills/C-Sharp.png"
+                  width="150px"
+                  height="150px"
```

Comment:
I think you used to use number in another component, consistency is a perfect approach for maintainable code

```suggestion
                  width={150}
                  height={150}
```